.cirrus.yml

```diff
@@ -70,16 +70,8 @@ task:
     memory: 1G
   # For faster CI feedback, immediately schedule the linters
   << : *CREDITS_TEMPLATE
-  setup_script:
-    - set -o errexit; source ./ci/test/00_setup_env.sh
-  before_install_script:
-    - set -o errexit; source ./ci/test/03_before_install.sh
-  install_script:
-    - set -o errexit; source ./ci/lint/04_install.sh
-  before_script:
-    - set -o errexit; source ./ci/lint/05_before_script.sh
   lint_script:
-    - set -o errexit; source ./ci/lint/06_script.sh
+    - ./ci/lint_run_all.sh
```

Comment:
I understand that it follows the way how we use the `./ci/test_run_all.sh` script, but it seems more readable and maintainable to do not introduce a second-level script:
```
  lint_script: |
    set -o errexit
    source ./ci/test/00_setup_env.sh
    source ./ci/test/03_before_install.sh
    source ./ci/lint/04_install.sh
    source ./ci/lint/05_before_script.sh
    source ./ci/lint/06_script.sh
```

Reply:
The script makes it also easier to run locally with just one command